python/unblob/handlers/filesystem/yaffs.py

```diff
@@ -8,8 +8,8 @@
 
 import attrs
 from structlog import get_logger
-from treelib import Tree
-from treelib.exceptions import NodeIDAbsentError
+from treelib import Tree  # type: ignore
+from treelib.exceptions import NodeIDAbsentError  # type: ignore
```

Comment:
I don't see this import flagged in the ci run

Reply:
fixed